### ISSUE TITLE
Smartdots .exe on github blocked by firewall

### ISSUE THREAD

**Issue by wimallegaert** [ices-eg/SmartDots#65]
Nabeil Salama from Scotland can't install SmartDots ui on his laptop. Dowloading/installing the .exe file is blocked by his firewall.

**Comment by wimallegaert** [comment]
We've suggested to whitlist the Github ip-addressess but i'm afraid that this is not a definitive solution.

**Comment by colinpmillar** [comment]
It might be informative to contact someone from the Scottish Government IT department.  Other quick fixes that have been employed by Scottish government scientists has been to run executables (such as R) from pendrives!  

I am visiting the Scottish lab next week and can ask around for potential solutions if that helps?

**Comment by colinpmillar** [comment]
@wimallegaert there are several others using smartdots at the same lab, might be worth asking them how they have done it

**Comment by wimallegaert** [comment]
@colinpmillar thanks for the additional information. I'll contact Nabeil Salama for an update.

**Comment by JCoadDavies** [comment]
Is this problem now fixed with the portable version that is available on github and which is available under the GETTING STARTED link on the SmartDots webpage?

**Comment by JCoadDavies** [comment]
Comment from ILVO at WGSMART WK Dec 2018

The   portable version can be a work-around, but with the portable version the   users will not be notified of new releases .     1 / Buy & install code signing certificate      2 / Change the automatic check for updates functionalitiy      + Replace the ClickOnce update check with a custom build check (cfr updates   Microsoft PowerBi). First we'll test if the firewall notification disappears   if we disable the click-one automatic update functionality. If so, the   automatic check for updates when SmartDots is launched will be removed.     + A custom build update check will be developed:         - We'll compare the installed   version with the most version. Information about the most recent version will   be stored on GitHub.          - If there is an update available   the user will be notified with a message in popup window, a link to the new   version will be displayd.          - If there is no update available   there will be no notification at all.
--

**Comment by wimallegaert** [comment]
Indeed, the portable version or the  code signing certificate (#64) should be a solution for this issue.
